javascript/javascript1/week3/ex-2.js

```diff
@@ -0,0 +1,16 @@
+//  Exercise 2: When will we be there??
+
+const travelInformation = {
+	speed: 50,
+	destinationDistance: 432,
+};
+
+function calculateTravelTime(travel) {
+	const totalTravelMinutes = (travel.destinationDistance / travel.speed) * 60;
```

Comment:
Very nice work; especially the variable naming! 🤩 

To be a bit annoying, if something took a minute, it would return "Travel time is: 0 hours and 1 minutes". Can you think of ways to improve this? 😄